source/tyr/tyr/tasks.py

```diff
@@ -455,8 +455,9 @@ def cities(osm_path):
         )
         if res != 0:
             logging.error('cities failed')
-    except:
-        logging.exception('')
+    except Exception as e:
+        logging.exception('cities exception : {}'.format(e.message))
```

Comment:
`logging.exception` already log the exception

Reply:
log without e.message:
[2018-10-03 11:32:03,037] [ERROR] [21584] [root] cities db not created
log with e.message:
[2018-10-03 11:32:38,547] [ERROR] [21637] [root] cities db not created (psycopg2.OperationalError) FATAL:  database "cities" does not exist